src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  margin: 0 8px;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.form[data-qa="big"] {
+  font-size: 16px;
+}
+
+.form[data-qa="small"] {
+  font-size: 14px;
+}
+
+.form-field {
+  width: 100%;
+  display: block;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  transition: 0.5s ease;
+  color: #3d4e61;
+  font-size: 1em;
+  line-height: 1.36em;
+  box-sizing: border-box;
+}
+
+.form-field::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.form-field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-field:focus {
+  border-radius: 3px;
+  outline: none;
+  box-shadow: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 800;
+}
+
+.search-big {
+  position: relative;
+  --big-icon-size: 19px;
+}
+
+.search-small {
+  position: relative;
+  --small-icon-size: 11px;
+}
+
+.search-big::before {
+  display: block;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  content: "";
+  width: var(--big-icon-size);
+  height: var(--big-icon-size);
+  background: url(./images/Search.svg) center center/cover no-repeat;
+}
+
+.search-big .form-field {
+  padding: 24px 25px 25px 62px;
+  height: 70px;
+}
+
+.search-small::before {
+  display: block;
+  content: "";
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: var(--small-icon-size);
+  height: var(--small-icon-size);
+  background: url(./images/Search.svg) center center/cover no-repeat;
+}
```

Comment:
looks like you repeat the same styles in both forms,  if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.